frontend/benefit/applicant/src/components/applications/forms/application/step2/ApplicationFormStep2.tsx

```diff
@@ -84,6 +84,7 @@ const ApplicationFormStep2: React.FC<DynamicFormStepComponentProps> = ({
   useAlertBeforeLeaving(formik.dirty);
 
   const selectLabel = t('common:select');
+  const hasCommissionOption = false;
```

Comment:
I suggest having a dedicated file for disabling/enabling features (e.g. [featureFlags.ts](https://github.com/City-of-Helsinki/berth-reservations-ui/blob/master/src/common/utils/featureFlags.ts)) and importing these booleans from it.

Reply:
god point